orchestrator/orchestrator.go

```diff
@@ -49,7 +51,10 @@ func NewOrchestrator(log log.Logger, closeApp context.CancelCauseFunc, networkCo
 	// Sping up OpSim to fornt the L2 instances
 	for i := range networkConfig.L2Configs {
 		cfg := networkConfig.L2Configs[i]
-		l2OpSims[cfg.ChainID] = opsimulator.New(log, closeApp, nextL2Port, l1Anvil, l2Anvils[cfg.ChainID], l2Anvils, networkConfig.InteropDelay)
+		if cfg.Host == "" {
+			cfg.Host = defaultHost
+		}
```

Comment:
by default the cli flags will fall back to 127.0.0.1 so this can be removed, no need to have `defaultHost`